internal/cli/atlas/security/ldap/save.go

```diff
@@ -102,15 +102,15 @@ func (opts *SaveOpts) validate() error {
 func (opts *SaveOpts) newLDAPConfiguration() *atlas.LDAPConfiguration {
 	ldapConfig := &atlas.LDAPConfiguration{
 		LDAP: &atlas.LDAP{
-			AuthenticationEnabled: opts.authenticationEnabled,
-			AuthorizationEnabled:  opts.authorizationEnabled,
-			Hostname:              opts.hostname,
-			Port:                  opts.port,
+			AuthenticationEnabled: &opts.authenticationEnabled,
+			AuthorizationEnabled:  &opts.authorizationEnabled,
+			Hostname:              &opts.hostname,
+			Port:                  &opts.port,
 			UserToDNMapping:       []*atlas.UserToDNMapping{},
-			BindUsername:          opts.bindUsername,
-			BindPassword:          opts.bindPassword,
-			CaCertificate:         opts.caCertificate,
-			AuthzQueryTemplate:    opts.authzQueryTemplate,
+			BindUsername:          &opts.bindUsername,
+			BindPassword:          &opts.bindPassword,
+			CaCertificate:         &opts.caCertificate,
+			AuthzQueryTemplate:    &opts.authzQueryTemplate,
```

Comment:
same for this. We have upgraded the go client in master and added other fixes as well.

Reply:
Done